fabric/src/main/java/muramasa/antimatter/integration/top/fabric/EnergyInfoProvider.java

```diff
@@ -0,0 +1,54 @@
+package muramasa.antimatter.integration.top.fabric;
+
+import mcjty.theoneprobe.api.*;
+import mcjty.theoneprobe.config.Config;
+import muramasa.antimatter.Ref;
+import muramasa.antimatter.capability.machine.MachineEnergyHandler;
+import muramasa.antimatter.tile.TileEntityMachine;
+import muramasa.antimatter.util.Utils;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.entity.BlockEntity;
+import net.minecraft.world.level.block.state.BlockState;
+
+public class EnergyInfoProvider implements IProbeInfoProvider {
+    @Override
+    public ResourceLocation getID() {
+        return new ResourceLocation(Ref.ID + ":energy_info");
+    }
+
+    @Override
+    public void addProbeInfo(ProbeMode mode, IProbeInfo probeInfo, Player player, Level level,
+                             BlockState blockState, IProbeHitData data) {
+
+        if (blockState.hasBlockEntity()) {
+            BlockEntity tile = Utils.getTile(level, data.getPos());
+
+            if (tile == null) return;
```

Comment:
This line is completely unnecessary, the instance check will return false if the tile is null.

Reply:
Removed